projects/view/client/projectProfile/projectProfile.html

```diff
@@ -5,50 +5,70 @@
   <!-- calling the helper function getProject that returns a project object using the route parameter projectId -->
     {{ #with getProject }}
 
+<div id="content-container">
     <div class="row">
-      <div class="col-md-6" style="text-align: center;">
+      <div id="project-media" class="col-md-6">
           {{#each projectMedia}}
             <div>
-              <img style="width:100%" src="{{this.url}}" alt="" />
+              <img id="project-media-img" src="{{this.url}}" alt="" />
+
+              <!-- Should be redone -->
+              <a href="#" onclick="showMoreImages();" id="project-image-version"> See more images</a>
             </div>
           {{/each}}
       </div>
       <div class="col-md-6">
-        <div class="page-header">
+        <div id="project-name" class="page-header">
           <h1>{{name}}</h1>
         </div>
-        <label>Progress: </label>
-        <div class="progress">
-          <div class="progress-bar" role="progressbar" aria-valuenow="{{calculateProgress}}" aria-valuemin="0" aria-valuemax="100" style="width: {{calculateProgress}}%;">
-            <span class="sr-only">{{calculateProgress}}% Complete</span>
-          </div>
-        </div>
-        <label>Launched in: </label><span>{{startupDateFormatted}}</span><br/>
-        <label>Based in: </label><span>{{postalAddress.country}}</span><br/>
-        <label>Website: </label>
-        {{# if hasMultipleWebsites }}
-          {{# each websites }}
-            <span class="label label-default {{typeToClass}}"><a href="{{url}}">{{name}}</a></span>
-          {{/ each }}
-        {{else}}
-          {{# each websites }}
-            <span class="label label-default"><a href="{{url}}">{{url}}</a></span>
-          {{/ each }}
-        {{/ if}}
+        <div id="project-data">
+          <label>Progress: </label>
```

Comment:
`<label>` tags are used when there is a form input. Consider using a different element, such as `<span>`.